### DISCUSSION TITLE
Create a Dashboard webpage for AirBnB and/or Swimply Guest

### DISCUSSION THREAD

**Discussion by spifftek70** [tagyoureit/nodejs-poolController#793] [Ideas]
Create a simplified Dashboard webpage for AirBnB and/or Swimply guests that does not give them access to the pool settings or configurations.  It should simply allow the guests to:

- turn on and off lights
- turn on and off waterfalls, aerators
- turn on the hot tub (jets, blowers, etc)
- set water temps

Make their access configurable on the main Dashboard page by the homeowner/admin.

**Comment by tagyoureit** [comment]
This has been on my [wishlist](https://github.com/tagyoureit/nodejs-poolController/discussions/610) for a while.  A few folks have asked for something like this over the years but I've been swamped.  Need to come back to this at some point, but not ETA from me right now.

**Reply by spifftek70** [reply]
So I created something to this, but specifically for my pool.  I'm not great at NodeJS, but I get by.  More of a PHP / Python guy myself.
Take a look at it and perhaps it can be a good starting point?

**Reply by tagyoureit** [reply]
Ok, thanks for taking a stab at this.  I'll take a look at some point, but I'm pretty swamped at the moment.

**Reply by spifftek70** [reply]
No worries. It would help a bunch to have access to the websockets to get pump status, temps, etc... That's really what I need help on. Right now, it's done doing Ajax, so not that efficient.
